packages/inter-protocol/test/psm/test-psm.js

```diff
@@ -338,6 +338,27 @@ test('limit', async t => {
   // t.throwsAsync(() => await E(seat1).getOfferResult());
 });
 
+test('limit is for minted', async t => {
+  const { minted, anchor } = t.context;
+  // only 50 minted allowed per anchor
+  const anchorPerMinted = makeRatio(50n, anchor.brand, 100n, minted.brand);
+  const driver = await makePsmDriver(t, { anchorPerMinted });
+
+  trace('test going over limit');
+  const giveTooMuch = anchor.make(MINT_LIMIT);
+  const seat1 = await driver.swapAnchorForMintedSeat(giveTooMuch);
+  t.throwsAsync(() => E(seat1).getOfferResult(), {
+    message: 'Request would exceed mint limit',
+  });
+  trace('limit is enforced on the Minted rather than Anchor');
+
+  trace('test right at limit');
+  const give = anchor.make(MINT_LIMIT / 2n);
+  const paymentPs = await driver.swapAnchorForMinted(give);
+  await paymentPs;
+  trace('test at minted limit works');
```

Comment:
```suggestion
  await t.notThrowsAsync(paymentPs, 'test that minted limit works');
```

at least spell out "that" :)

Reply:
I feel more strongly about this being an actual test statement so included it in my requested changes